doc/releases/reef.rst

```diff
@@ -5,6 +5,417 @@ Reef
 Reef is the 18th stable release of Ceph. It is named after the reef squid
 (Sepioteuthis).
 
+v18.2.3 Reef
+============
+This is the third backport release in the Reef series. We recommend that all users update to this release.
```

Comment:
Add a blank line above.

Reply:
FWIW it was added in the wrong place -- a blank line is needed after `============` and `This is the third backport ...` lines.

Reply:
done